crates/rolldown_binding/src/options/plugin/parallel_js_plugin.rs

```diff
@@ -139,23 +139,25 @@ impl Plugin for ParallelJsPlugin {
   async fn generate_bundle(
     &self,
     ctx: &rolldown_plugin::SharedPluginContext,
-    bundle: &Vec<rolldown_common::Output>,
+    bundle: Vec<rolldown_common::Output>,
     is_write: bool,
-  ) -> rolldown_plugin::HookNoopReturn {
+  ) -> rolldown_plugin::HookGenerateBundleReturn {
     if self.first_plugin().generate_bundle.is_some() {
-      self.run_all(|plugin| plugin.generate_bundle(ctx, bundle, is_write)).await?;
+      self.run_single(|plugin| plugin.generate_bundle(ctx, bundle, is_write)).await
+    } else {
+      Ok(vec![])
     }
-    Ok(())
   }
 
   async fn write_bundle(
     &self,
     ctx: &rolldown_plugin::SharedPluginContext,
-    bundle: &Vec<rolldown_common::Output>,
-  ) -> rolldown_plugin::HookNoopReturn {
+    bundle: Vec<rolldown_common::Output>,
```

Comment:
Want's wrong with using `&mut Vec<rolldown_common::Output>`?

Reply:
It should be correct. Let me update code.